examples/View.elm

```diff
@@ -95,16 +95,33 @@ errorNotif error =
 
 
 view : Model -> Html.Html Msg
-view { error, records, formData, currentTime, sort } =
-    Html.div [ Html.Attributes.class "container" ]
-        [ Html.h1 [] [ Html.text "elm-kinto demo" ]
-        , Html.small
-            []
-            [ Html.text "Will only display 30 records at most" ]
-        , errorNotif error
-        , recordsList records currentTime sort
-        , formView formData
-        ]
+view { error, records, formData, currentTime, sort, limit } =
+    let
+        lim =
+            case limit of
+                Just limit ->
+                    toString limit
+
+                Nothing ->
+                    ""
+    in
+        Html.div [ Html.Attributes.class "container" ]
+            [ Html.h1 [] [ Html.text "elm-kinto demo" ]
+            , Html.small
+                []
+                [ Html.text "Only display "
+                , Html.input
+                    [ Html.Attributes.value lim
+                    , Html.Events.onInput NewLimit
```

Comment:
So this actually performs a HTTP query everytime the vaue is updated. I'd personally rather see a submit button so the users can press it when they're happy with their entered number.